recipes/r-saige/conda_build_config.yaml

```diff
@@ -1,4 +1,4 @@
 r_base:
-  - 3.5.1
   - 3.6.1
-  - 4.0
+  # NOTE: Adding ' *' to workaround https://github.com/conda/conda-build/issues/3974.
+  - '4.0 *'
```

Comment:
Due to a [`conda-build` bug](https://github.com/conda/conda-build/issues/3974) we can't use `4.0` (already in `conda-forge-pinning`) or `4.0.*` (already in `bioconda-utils_conda_build_config.yaml`) here.
I hope this doesn't mess anything else up. Will definitely have to check the dep constraints in the artifacts before merging!

Reply:
@mbargull thanks, I looked in the log and it seems to work well

Reply:
@mbargull although this fails on macos
```
23:43:12 BIOCONDA INFO (OUT) Error in dyn.load(file, DLLpath = DLLpath, ...) :
23:43:12 BIOCONDA INFO (OUT)   unable to load shared object '$PREFIX/lib/R/library/Rcpp/libs/Rcpp.dylib':
23:43:12 BIOCONDA INFO (OUT)   dlopen($PREFIX/lib/R/library/Rcpp/libs/Rcpp.dylib, 6): Symbol not found: _EXTPTR_PTR
23:43:12 BIOCONDA INFO (OUT)   Referenced from: $PREFIX/lib/R/library/Rcpp/libs/Rcpp.dylib
23:43:12 BIOCONDA INFO (OUT)   Expected in: /Users/
23:43:12 BIOCONDA INFO (OUT) Calls: <Anonymous> ... asNamespace -> loadNamespace -> library.dynam -> dyn.load
23:43:12 BIOCONDA INFO (OUT) Execution halted
```

the difference is that:
- with `- '4.0 *'` it applies `r-base: 4.0.0-hec7d121_3`
- with `- 4.0` or `- 4.0.*` it applies `r-base: 4.0.3-hd272fe0_2`

do you think it would work if I try `- '4.0.* *'`? I guess the trick is to get it to use `4.0.3`

Reply:
Ah, yes, that was an oversight on my part, thanks! We usually want `X.Y.*` but `conda-build` also changes `X.Y` to `X.Y.*` automatically (convenience and IIRC also due to historic reasons) -- but only if `.Y` is the suffix, hence not with the `.Y *` suffix.